source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -165,81 +220,152 @@ Table 3.p (below): List of Data_Type Values
+     - A&I_Database\ |br|\ |lb|
+       Aggregated_Full_Content\ |br|\ |lb|
+       Discovery_Service\ |br|\ |lb|
+       eJournal\ |br|\ |lb|
+       Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, IR
 
    * - Newspaper_or_Newsletter
      - Textual content published serially in a newspaper or newsletter.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
-       eJournal\ |br|\ |lb|
-       Repository
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1
+       eJournal
+     - PR, DR, TR
 
    * - Other
-     - Content that cannot be classified by any of the other Data_Types.
-
-       Note that Data_Type Other MUST NOT be used if there isn’t sufficient information available to classify the content.
+     - Content that has been labelled with a data type that does not exist within and cannot be mapped to COUNTER’s Code of Practice. Other MUST NOT be used if there is not sufficient information available to classify the content.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
        Repository
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1
+     - PR, DR, TR, IR
+
+   * - Patent
+     - A patent document representing an exclusive right granted for an invention, which is a product or a process that provides, in general, a new way of doing something, or offers a new technical solution to a problem. Typically associated with a patent number.
+     - Aggregated_Full_Content\ |br|\ |lb|
+       Discovery_Service\ |br|\ |lb|
+       Full_Content_Database\ |br|\ |lb|
+       Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, DR, TR, IR
 
    * - Platform
-     - A content platform that may reflect usage from multiple Data_Types. Note that Data_Type Platform is only applicable for Searches_Platform.
+     - *Only applies to Searches_Platform metrics.*
      - All Host_Types
-     - PR\ |br|\ |lb|
-       PR_P1
+     - PR
+
+   * - Reference_Item
+     - An item or record within a reference work (e.g. an encylopedia reference), or a reference item available as a standalone content item not aggregated into a title, for example in an institutional repository. 
+
+       Where a whole reference work is being downloaded and there is no mechanism to determine the number of items, one Reference_Item should be reported in the IR.
+     - Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, IR
+
+   * - Reference_Work
+     - An authoritative source of information about a subject used to find quick answers to questions, such as an encyclopedia or dictionary. The content may be stable or updated over time.
+     - Aggregated_Full_Content\ |br|\ |lb|
+       Discovery_Service\ |br|\ |lb|
```

Comment:
Patent and Reference_Work are the only two Data_Types where Discovery is listed, but not A&I_Database. I'm wondering if that is correct?